packages/react-tree-grid/package.json

```diff
@@ -5,6 +5,7 @@
   "peerDependencies": {
     "@fluentui/react-components": ">=9.44.4 < 10.0.0",
     "@fluentui/react-shared-contexts": ">=9.7.2 <10.0.0",
```

Comment:
This package package should not be defined in `peerDependencies` -  all bits should be coming from `@fluentui/react-components`. Anyway, I don't see any usage of this package anymore.